yellowbrick/classifier/prcurve.py

```diff
@@ -458,10 +469,17 @@ def precision_recall_curve(model, X, y, ax=None, train_size=0.8,
     # Instantiate the visualizer
     viz = PRCurve(model, ax=ax, **kwargs)
 
-    # Create train and test splits to validate the model
-    X_train, X_test, y_train, y_test = tts(
-        X, y, train_size=train_size, random_state=random_state, shuffle=shuffle
-    )
+
+    if (X_test is None) and (y_test is None):
```

Comment:
We should probably test the case where `(X_test is not None and y_test is None) or (X_test is None and y_test is not None)` and raise a `YellowbrickValueError("both X_test and y_test are required if one is specified")`. Otherwise, this will raise an exception in `tts`. 

Are there any other conditions that might cause problems?

Reply:
Any other exception would be raised by tts i'm guessing. I think we're good with these tests.